### ISSUE TITLE
Add 2 more Subscribe buttons

### ISSUE THREAD

**Issue by rclabough** [LemmyNet/lemmy-ui#1082]
For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy)

### Is your proposal related to a problem?

I think we should add more subscribe buttons for communities outside of the local server.  For a user to subscribe to another instance's community,
1) Search for that community
2) Click the community
3) Open sidebar (if on mobile)
4) Click Subscribe

This is confusing to most users

### Describe the solution you'd like

![image](https://github.com/LemmyNet/lemmy-ui/assets/6788536/8ed95844-220d-405f-84bc-fb3656e01a69)
![image](https://github.com/LemmyNet/lemmy-ui/assets/6788536/52939f25-293f-4004-94bb-a32377ef9a7d)
![image](https://github.com/LemmyNet/lemmy-ui/assets/6788536/e30096ce-f9d0-457b-8361-71bc02096a4c)

Could we get two new subscription links?  One on the search itself, next to the results there should be a link/button to subscribe right there.  

Second is if in mobile view, before [Expand Sidebar] there should be a subscribe button.

Subscribing to communities should be one of the easiest things on the platform, with the ability to subscribe pretty much everywhere it can be, that will drive user engagement with the platform.

Thanks for the hard work!

**Comment by dessalines** [comment]
I don't think we should add subscribe buttons to post listings or comment nodes, which are the items you linked. 

I would be open to moving the subscribe buttons out of the sidebar tho on mobile, as long as it isn't too obtrusive.

**Comment by rclabough** [comment]
At least out of the sidebar on mobile (or duplicated), these are images that users gave me and told me exactly "They couldn't figure out how to subscribe".  Obviously with one of them being there, in the image even.  As long as we're making it easier to subscribe I think that's moving forward
